### DISCUSSION TITLE
Is the package @undp/carbon-services-lib used in backend/services also open source?

### DISCUSSION THREAD

**Discussion by eva-devs** [undp/carbon-registry#253] [General]
The package @undp/carbon-services-lib is loaded in the backend/service package.json. It holds a lot of important libraries.

We are just evaluating this project as a possible base for a registry in Germany, but without the source code of these libraries it's hard to evaluate the real implementation.

Are there any plans to publish the source code?

@mbelinsky @palindaa

**Comment by palindaa** [comment]
It is an open source repository https://github.com/undp/carbon-library. This repository includes  @undp/carbon-services-lib and @undp/carbon-library source code. Will update the documentation on that.

**Reply by eva-devs** [reply]
Thanks a lot!
